src/platform/requestLogger/node/requestLogger.ts

```diff
@@ -77,13 +91,15 @@ export interface ILoggedElementInfo {
 	maxTokens: number;
 	trace: HTMLTracer;
 	chatRequest: ChatRequest | undefined;
+	toJson(): any;
```

Comment:
can you just use `object` instead of `any` for all these return types? (in the impl file as well)